### ISSUE TITLE
Aggregator App : Spring Integration header ID is required for persistence with REDIS

### ISSUE THREAD

**Issue by CEDDM** [spring-cloud-stream-app-starters/aggregator#29]
Hi all
When setting up persistence with REDIS, I get the error "Cannot store messages without an ID header". To resolve this, I have to manually add the property "spring.cloud.stream.kafka.default.consumer.standard-headers = both" to map the ID and TIMESTAMP headers of Spring Integration in Spring Kafka. As the Spring Integration header ID is required for persistence with REDIS, could this property be added automatically by the app (I looked for a while before finding the solution!)?
The version I am using is “2.4.0.RELEASE”.

**Comment by artembilan** [comment]
Yes, we can do that as out-of-the-box feature providing the mentioned property as part of generated app via appropriate Maven plugin configuration in this project.
Something like:
```
<additionalAppProperties>
    <additionalAppProperty>spring.cloud.stream.kafka.default.consumer.standard-headers = both</additionalAppProperty>
</additionalAppProperties>
```
